staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -69,6 +70,14 @@ var (
 		},
 		[]string{"resource"},
 	)
+	etcdLeaseObjectCounts = compbasemetrics.NewGaugeVec(
+		&compbasemetrics.GaugeOpts{
+			Name:           "etcd_lease_object_counts",
+			Help:           "Number of objects in last reused etcd lease.",
```

Comment:
I'm wondering if we really want to store the "last lease".
Maybe we want a histogram? Or maybe just max?

I'm not convinced the "last" metric is the most interesting one.

Reply:
done